storage/mysql/log_storage.go

```diff
@@ -296,7 +278,7 @@ func (t *logTreeTX) DequeueLeaves(ctx context.Context, limit int, cutoffTime tim
 	}
 
 	leaves := make([]*trillian.LogLeaf, 0, limit)
-	dql := make([]*dequeuedLeaf, 0, limit)
+	dq := make([]interface{}, 0, limit)
```

Comment:
Not sure why this changed to interface{}, seems very generic. Possibly make dequeuedLeaf a storage level type to get round it? or is that no good?